model/import/CsvImportForm.php

```diff
@@ -94,7 +94,7 @@ public function initElements()
         );
 
         $this->form->addElement($formElement);
-        $downloadLink = '#';
+        $downloadLink = '/taoQtiItem/QtiCreator/downloadCsv?uri='.urlencode($_POST['classUri']);
```

Comment:
Could you avoid hardcoded `urls` by using `\oat\tao\helpers\UrlHelper::buildUrl`, which would help you to build urls.
Also please do not use directly data from global variables (e.g. $_POST['classUri']) , as that may introduce some security risks

Reply:
It was done